src/signTx.h

```diff
@@ -49,12 +49,20 @@ enum {
 	SIGN_MAX_WITNESSES = SIGN_MAX_INPUTS + SIGN_MAX_OUTPUTS + SIGN_MAX_CERTIFICATES + SIGN_MAX_REWARD_WITHDRAWALS,
 };
 
+typedef struct {
```

Comment:
I'd rename the fields here.
`pathSet` -> `isPathSet`
`byron` -> `isByron`
Similarly in `violatesAndStoreSingleAccount` then.

Reply:
The conventions are more akin to my version I think, like `bool auxDataReceived`. I'm on the fence with isByron.